code/modules/clothing/clothing.dm

```diff
@@ -792,3 +792,10 @@ BLIND     // can't see anything
 		deconstruct(FALSE)
 	else
 		..()
+
+/obj/item/clothing/neck
+	name = "necklace"
+	icon = 'icons/obj/clothing/neck.dmi'
+	body_parts_covered = UPPER_TORSO
+	slot_flags = SLOT_NECK
+	strip_delay = 40
```

Comment:
Use the SECONDS define here please. `4 SECONDS`. I would suggest changes but im on a mobile.